CHANGES.md

```diff
@@ -40,6 +40,11 @@
 
 - Lines ending with `fmt: skip` will now be not formatted (#1800)
 
+- PR #2053: Black no longer relies on typed-ast for Python 3.8 and higher
+- PR #2053: Python 2 support is now optional, install with the `python2` extra to
+  maintain support
+  [pip extras #7](https://pip.pypa.io/en/stable/reference/pip_install/#examples)
```

Comment:
1) I'm surprised prettier isn't ticked off here by the lack of spacing; and

2) if you'd like, just use your original suggestion of `python3 -m pip install typed_ast` since this reads worse and my reasoning was at best questionable.